### PR TITLE
Update Spanish to ver.23

### PR DESCRIPTION
**Pull request by xa31er**

### CODE REVIEW

File: translation/es-ES.toml [hunk 1]
```diff
@@ -1,27 +1,35 @@
 name = "Español" # Name of language
 direction = "ltr" # Change to "rtl" for right to left languages
-version = "21"
+version = "23"
 maintainer = "xa31"
 
 [header]
 home = "Inicio"
```
Comment (Naviary2) [inline translation/es-ES.toml:7]: The value of this in update 22 was changed from "Home" to "Infinite Chess" in the english TOML. Does this need to change?

File: translation/es-ES.toml [hunk 2]
```diff
@@ -1,27 +1,35 @@
...
+language = "Idioma"
+board = "Tablero" # Board color/theme
+legalmoves = "Jugadas Legales" # Legal moves shape
+legalmoves-squares = "Casillas"
+legalmoves-dots = "Puntos" # Dots and 4 corner triangles
+perspective = "Perspectiva" # Perspective-mode
+perspective-mouse-sensitivity = "Sensibilidad del ratón"
+perspective-fov = "Campo de visión (FOV)"
+ping = ["Latencia", "ms"] # A number is inserted between these 2 strings.
+reset-to-default = "Reiniciar opciones por defecto"
```
Comment (Naviary2) [inline translation/es-ES.toml:25]: I don't think the "Reset to default" button will have enough space for this in the dropdown, or the button will be MASSIVE. Can this be simplified?
Screenshot 2024-12-06 at 1 18 54 PM

**Comment by Naviary2** [comment]
index.how_to_paragraph also was updated to reflect the latest version of the game, in TOML update No. 23

**Comment by Naviary2** [comment]
Just those 3 things were what I found!

File: translation/es-ES.toml (hunk 1 above)
Comment (xa31er) [reply translation/es-ES.toml:7]: Oh, sorry, I missed that.

File: translation/es-ES.toml (hunk 2 above)
Comment (xa31er) [reply translation/es-ES.toml:25]: I'll look at it

**Comment by xa31er** [comment]
I really cannot make the "reset to defaults" sentence any shorter.

**Comment by Naviary2** [comment]
Thank you!

**Pull request merged**